jina/serve/gateway.py

```diff
@@ -22,28 +23,30 @@ class BaseGateway(JAMLCompatible):
 
     def __init__(
         self,
+        args: 'argparse.Namespace',
```

Comment:
we can't add this here and I actually don't see why it needs to be added, it's only needed in set_streamer and is already passed as parameter